Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -160,7 +172,6 @@ - (void)handleRemoteNotificationsRegistered:(NSNotification *)notification
   if ([app respondsToSelector:@selector(registerUserNotificationSettings:)]) {
     UIUserNotificationSettings *notificationSettings = [UIUserNotificationSettings settingsForTypes:(NSUInteger)types categories:nil];
     [app registerUserNotificationSettings:notificationSettings];
-    [app registerForRemoteNotifications];
```

Comment:
Doesn't removing this prevent the application:didRegisterForRemoteNotificationsWithDeviceToken: or application:didFailToRegisterForRemoteNotificationsWithError: callbacks from being called?